copy/copy.go

```diff
@@ -1060,6 +1060,14 @@ func (ic *imageCopier) copyLayer(ctx context.Context, srcInfo types.BlobInfo, to
 			logrus.Debugf("Skipping blob %s (already present):", srcInfo.Digest)
 			bar := ic.c.createProgressBar(pool, srcInfo, "blob", "skipped: already exists")
 			bar.SetTotal(0, true)
+
+			// Throw an event that the layer has been skipped
+			if ic.c.progress != nil {
```

Comment:
The other condition has a `&& c.progressInterval > 0`. Is there a reason to deviate?

Reply:
Hm, I added the check for sake of consistency in behavior. :+1: